docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,25 @@
 Version history
 ===============
 
+5.3.0-m2 (2016 November 23)
+---------------------------
+
+Fixes and clean up:
+
+* clean up of bundles, packages and build targets to reflect decoupling work
+* POM repositories clean up
+* updated developer documentation on building Bio-Formats
+* updated testing packages
+* updated build versioning from Maven
+
+Further decoupling:
+
+* removed stubs and metakit and replaced them with jars from Maven Central
+
+File format support:
+
+* added support for JPEG-XR compressed CZI data
```

Comment:
I would minimally mention `ome:jxrlib` as a new dependency of Bio-Formats. Is it worth linking to the blog post/press announcement? @melissalinkert @chris-allan : anything in particular to mention on this page or should we generically review as part of the documentation of the CZI work?